examples/group_vars/setup_vars

```diff
@@ -0,0 +1,107 @@
+# TODO: Make hostname a variable instead of the whole API link
+dr_sites_primary_url: "http://mlipchuk.usersys.redhat.com:8080/ovirt-engine/api"
+dr_sites_primary_username: "admin@internal"
+dr_sites_primary_ca_file: "/home/mlipchuk/ovirt-share/etc/pki/ovirt-engine/ca.pem"
+
+dr_sites_secondary_url: "http://10.35.0.140:8080/ovirt-engine/api"
+dr_sites_secondary_username: "admin@internal"
+dr_sites_secondary_ca_file: "/home/mlipchuk/ovirt-share/etc/pki/ovirt-engine/ca.pem"
+
+dr_import_storage:
+    - {dr_domain_type: "nfs",
+       dr_master_domain: "True",
+       dr_primary_address: "10.35.16.43",
+       dr_primary_path: "/export/data_test7",
+       dr_primary_dc_name: "Prod",
+       dr_primary_name: "data7",
+
+       dr_secondary_address: "10.35.1.68",
+       dr_secondary_path: "/home/export/data2",
+       dr_secondary_dc_name: "Recovery",
+       dr_secondary_name: "data2"}
+
+    - {dr_domain_type: "nfs",
+       dr_master_domain: "False",
+       dr_primary_name: "data6",
+       dr_primary_address: "10.35.16.43",
+       dr_primary_path: "/export/data_test6",
+       dr_primary_dc_name: "Prod",
+
+       dr_secondary_name: "data",
+       dr_secondary_address: "10.35.1.68",
+       dr_secondary_path: "/home/export/data1",
+       dr_secondary_dc_name: "Recovery"}
+
+    - {dr_domain_type: "iscsi",
+       dr_master_domain: "False",
+       dr_primary_id: "aa92cc71-1b88-4998-a755-970ef8a638ea",
+       dr_primary_address: "10.35.16.55",
+       dr_primary_target: ["iqn.2015-07.com.mlipchuk1.redhat:444"],
+       dr_primary_username: "",
+       # TODO: Passwords should be encapsulated
+       dr_primary_password: "",
+       dr_primary_port: "",
+       dr_primary_dc_name: "Prod",
+       dr_primary_name: "scsi2",
+
+       dr_secondary_id: "ccccf47b-ff77-4660-bf07-2809a750adb6",
+       dr_secondary_address: "10.35.0.222",
+       dr_secondary_target: ["iqn.2015-07.com.recovery.redhat:444"],
+       dr_secondary_username: "",
+       # TODO: Passwords should be encapsulated
+       dr_secondary_password: "",
+       dr_secondary_port: "",
+       dr_secondary_dc_name: "Recovery",
+       dr_secondary_name: "scsi2"}
+
+# Mapping for cluster
+dr_cluster_mapping:
```

Comment:
I think it's better to use plural for list variables

Reply:
done, for all of the mappings